pr_agent/algo/pr_processing.py

```diff
@@ -78,9 +78,9 @@ def get_pr_diff(git_provider: GitProvider, token_handler: TokenHandler, model: s
     return final_diff
 
 
-def pr_generate_extended_diff(pr_languages: list, token_handler: TokenHandler,
-                              add_line_numbers_to_hunks: bool) -> \
-        Tuple[list, int]:
+def pr_generate_extended_diff(pr_languages: list,
+                              token_handler: TokenHandler,
+                              add_line_numbers_to_hunks: bool) -> Tuple[list, int, list]:
```

Comment:
**Suggestion:** Consider adding a return type hint to the 'pr_generate_extended_diff' function to improve code readability and maintainability.
```suggestion
def pr_generate_extended_diff(pr_languages: list,
                              token_handler: TokenHandler,
                              add_line_numbers_to_hunks: bool) -> Tuple[List[str], int, List[int]]:
```